main.go

```diff
@@ -1590,9 +1628,11 @@ func computePkgCache(fsCache *cache.Cache, lpkg *listedPackage, pkg *types.Packa
 	if err := gob.NewEncoder(&buf).Encode(computed); err != nil {
 		return pkgCache{}, err
 	}
+
 	if err := fsCache.PutBytes(lpkg.GarbleActionID, buf.Bytes()); err != nil {
 		return pkgCache{}, err
 	}
+
```

Comment:
please avoid unnecessary whitespace or cleanup changes as part of this big refactor, to avoid noise